backend/services/location.service.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable func-names */
+const { Location } = require('../models/dictionaries/location.model');
+const DatabaseError = require('../errors/database.error');
+
+const LocationService = {};
+
+function validateLocations(locationsarray){
+
+  if(!Array.isArray(locationsarray)){
+    throw "locations array must be of type array"
+  }
+  if(locationsarray.length <= 0){
```

Comment:
please fix the condition. It should be `locationsarray.length === 0`

Reply:
condition fixed